File: garrafa.cs
public class Garrafa
{
    public int Altura { get; set; }
    public int Largura { get; set; }
    public int Capacidade { get; set; }

    public void Abrir() { }
    public void Fechar() { }

    private voide Encher(){
        while(Total < Capacidade)
            Total++;
    }
}

File: garrafa.dart
class Garrafa{
    int _total = 0;
    int altura = 0;
    int largura = 0;
    int comprimento = 0;

    void abrir() { }
    void fechar() { }   
    void encher(){
        while(_total < capacidade)
            _total++;
    }
}